### PR TITLE
Added LED Brightness Adjust

### PR DESCRIPTION
**Pull request by jpmreece**
The Configurator has been modified to allow for a bightness adjust adjrange. This corresponds to the pull request on Betaflight " LED Dimmer #13776"
https://github.com/betaflight/betaflight/pull/13776

This has been tested on Windows to program a SpeedyBeeF7V3

### CODE REVIEW

**Comment by nerdCopter** [comment]
Tested UI, but not functions.
 image

**Comment by jpmreece** [comment]
> Tested UI, but not functions. image

Does this mean that I need to do further testing? It should pass an adjrange through to the device.

**Comment by nerdCopter** [comment]
> Does this mean that I need to do further testing? It should pass an adjrange through to the device.

It only means i did not fully test and maybe do not know how to fully test and further do not have the hardware to fully test.
I put the screenshot so at least other's know _where_ to test.

**Comment by jpmreece** [comment]
> > Does this mean that I need to do further testing? It should pass an adjrange through to the device.
> 
> It only means i did not fully test and maybe do not know how to fully test and further do not have the hardware to fully test. I put the screenshot so at least other's know _where_ to test.

I tested (before testing with a drone) that saving added the appropriate adjrange value of 31. This can be checked in the cli and should correspond with the enum in rc_adjustments.h

`typedef enum {
    ADJUSTMENT_NONE = 0,
    ADJUSTMENT_RC_RATE,
    ADJUSTMENT_RC_EXPO,
    ADJUSTMENT_THROTTLE_EXPO,
    ADJUSTMENT_PITCH_ROLL_RATE,
    ADJUSTMENT_YAW_RATE,
    ADJUSTMENT_PITCH_ROLL_P,
    ADJUSTMENT_PITCH_ROLL_I,
    ADJUSTMENT_PITCH_ROLL_D,
    ADJUSTMENT_YAW_P,
    ADJUSTMENT_YAW_I,
    ADJUSTMENT_YAW_D,
    ADJUSTMENT_RATE_PROFILE,
    ADJUSTMENT_PITCH_RATE,
    ADJUSTMENT_ROLL_RATE,
    ADJUSTMENT_PITCH_P,
    ADJUSTMENT_PITCH_I,
    ADJUSTMENT_PITCH_D,
    ADJUSTMENT_ROLL_P,
    ADJUSTMENT_ROLL_I,
    ADJUSTMENT_ROLL_D,
    ADJUSTMENT_RC_RATE_YAW,
    ADJUSTMENT_PITCH_ROLL_F,
    ADJUSTMENT_FEEDFORWARD_TRANSITION,
    ADJUSTMENT_HORIZON_STRENGTH,
    ADJUSTMENT_ROLL_RC_RATE,
    ADJUSTMENT_PITCH_RC_RATE,
    ADJUSTMENT_ROLL_RC_EXPO,
    ADJUSTMENT_PITCH_RC_EXPO,
    ADJUSTMENT_PID_AUDIO,
    ADJUSTMENT_PITCH_F,
    ADJUSTMENT_ROLL_F,
    ADJUSTMENT_YAW_F,
    ADJUSTMENT_OSD_PROFILE,
    ADJUSTMENT_LED_PROFILE,
	ADJUSTMENT_LED_DIMMER,
    ADJUSTMENT_FUNCTION_COUNT
} adjustmentFunction_e;`

**Review by nerdCopter — APPROVED** [review]
- approving based on code review; could not test actual LED/transmitter-adjustments.
- depends on https://github.com/betaflight/betaflight/pull/13776

**Review by blckmn — APPROVED** [review]

File: src/js/tabs/adjustments.js [hunk 1]
```diff
@@ -272,7 +272,7 @@ adjustments.cleanup = function (callback) {
 
 adjustments.adjust_template = function () {
     const selectFunction = $('#functionSelectionSelect');
-    const elementsNumber = 31; // 'LED Profile Select' are the last adjustmentsFunction
+    const elementsNumber = 32; // 'LED Dimmer' are the last adjustmentsFunction
```
Comment (haslinghuis) [inline src/js/tabs/adjustments.js:275]:
```suggestion
    const elementsNumber = 32; // 'LED Dimmer' is the last adjustmentsFunction
```

**Review by haslinghuis — APPROVED** [review]

**Pull request merged**